tools/python/test_server/server/testserver.py

```diff
@@ -54,12 +55,6 @@
 
 logging.basicConfig(format='%(asctime)s %(message)s', level=logging.DEBUG)
 
-PORT = 34568
-LIFESPAN = 180.0  # timeout for the self destruction timer - how much time
```

Comment:
Why moving these vars into a config better than having them here?

Reply:
`PORT` var is required for `stop_server.py`

In this case it's better to move all configs into a separate file. + there are problems with python packages if we `import testserver.py` directly in `stop_server.py`